lucetc/src/load.rs

```diff
@@ -28,12 +22,11 @@ pub fn read_module<P: AsRef<Path>>(
 }
 
 pub fn read_bytes(bytes: Vec<u8>) -> Result<Vec<u8>, Error> {
-    let converted = if wasm_preamble(&bytes) {
-        bytes
+    if wasm_preamble(&bytes) {
+        Ok(bytes)
     } else {
         wat2wasm(bytes).map_err(|err| {
-            let mut result = String::from("wat2wasm error: ");
-            result.push_str(&format!("{}", err));
+            let mut result = format!("wat2wasm error: {}", err);
             match unsafe { std::mem::transmute::<wabt::Error, wabt::ErrorKind>(err) } {
```

Comment:
oh uh. this shouldn't be necessary anymore. I'll do a cleanup PR for this once this one lands.